packages/zowe-explorer/package.json

```diff
@@ -1567,6 +1567,12 @@
           "description": "Zowe-Temp-Folder-Loacation",
           "scope": "window"
         },
+        "Zowe-Hide-Temp-Folder": {
```

Comment:
@crawr Do we want to go ahead and try to use the [standardized VS Code formatting](https://code.visualstudio.com/api/references/contribution-points#Configuration-schema) for this setting? Something like `zowe.hideTempFolder`?

Reply:
Agree. Otherwise, user will start using it and then have to migrate again.

Reply:
Yes, this makes sense to me too. I've changed it in the latest commit